dsfhub/resource_cloud_account_test.go

```diff
@@ -42,6 +42,67 @@ func TestAccDSFCloudAccount_Aws(t *testing.T) {
 	})
 }
 
+func TestAccDSFCloudAccount_Azure(t *testing.T) {
+	gatewayId := os.Getenv("GATEWAY_ID")
+	if gatewayId == "" {
+		t.Skip("GATEWAY_ID environment variable must be set")
+	}
+
+	const (
```

Comment:
how would you feel about putting these consts in the test data file?

Reply:
In my view, the test_data files are essentially just for configuration constants/functions and the constants that pertain to the actual tests should be included in the test themselves as I think it makes them a little more readable (although I could probably be persuaded otherwise).